tests/unit_tests/shared/share/test_ecl_run.py

```diff
@@ -375,54 +370,31 @@ def test_run(init_ecl100_config, source_root):
         "SPE1.DATA",
     )
     econfig = ecl_config.Ecl100Config()
-    sim = econfig.sim("2014.2")
+    sim = econfig.sim("2019.3")
     erun = ecl_run.EclRun("SPE1.DATA", sim)
     erun.runEclipse()
 
     ok_path = os.path.join(erun.runPath(), f"{erun.baseName()}.OK")
-    log_path = os.path.join(erun.runPath(), f"{erun.baseName()}.LOG")
+    prt_path = os.path.join(erun.runPath(), f"{erun.baseName()}.PRT")
```

Comment:
What kind of extension is `prt`? I result of a search and chatGPT summarised in a way that does not make sense for me here 🤔 : 

> ".prt" (Part) file extension:
> CAD Software: In the field of computer-aided design (CAD), ".prt" files are often associated with 3D models created using software like PTC Creo, Siemens NX, or SolidWorks. These files contain the geometric and structural information of a part or component.
> Printer-related: ".prt" can also be used as an abbreviation for "print" in certain applications, where the file may contain instructions or settings specific to printing a document or image.

Reply:
ChatGPT is utterly wrong here.. PRT is a extension name chosen by the Eclipse simulator and is shorthand for "PRINT". We call the PRT-file for the PRINT-file from Eclipse, representing the same thing as a log file (why there is sometimes a LOG file as well I do not know the details of).

Reply:
Given the confusion I made, what do you think about renaming to `print_file_path` instead of `prt_file`?
But maybe that is obvious for people in general and then the comment can be considered irrelevant :-)

Reply:
I can remove, or at least comment. This is only obvious to Eclipse people, not to ERT-programmers.